hnn_core/pyramidal.py

```diff
@@ -334,62 +335,7 @@ def _get_soma_props(self, pos, p_all):
         }
 
     def secs(self):
```

Comment:
Do we need to define a `secs` method for each cell class anymore?

Reply:
Not sure but for now I would say yes. We would need a more direct way of populating it in the future though.

Reply:
Oh I see, because `secs` is called by the parent class.